docs/graphql/core/scheduled-triggers/create-one-off-scheduled-event.rst

```diff
@@ -131,3 +131,11 @@ Once you've created your scheduled trigger, you can see ``Pending events``, ``Pr
 .. thumbnail:: /img/graphql/core/event-triggers/pending-one-off.png
    :alt: Schedule and logs for scheduled events
    :width: 1200px
+
+
+One minute precision
+--------------------
+
+This means that for any scheduled event set at a particular time, it actually delivers on the 0th second of the next minute.
+
+For example: If you schedule an event to arrive at this particular time ``09:24:10``, because of the one minute precison, it will arrive at anytime between ``09:24:10`` and ``09:25:10`` max.
```

Comment:
Here we need to think about the wording a bit. We don't use "arrive" in the context of triggers anywhere else in the docs. Maybe we can use "deliver"?

Reply:
Okay